app/src/main/java/org/simple/clinic/bp/entry/BloodPressureEntryUpdate.kt

```diff
@@ -101,4 +118,14 @@ class BloodPressureEntryUpdate(
     return (systolicText.length == 3 && systolicText.matches("^[123].*$".toRegex()))
         || (systolicText.length == 2 && systolicText.matches("^[789].*$".toRegex()))
   }
+
+  private fun formatToPaddedDate(day: String, month: String, twoDigitYear: String, fourDigitYear: String): String {
```

Comment:
Ideally, this should be done in the UI and passed to the mobius loop as a date changed event with the padded string as the property, right?

Reply:
Yes, that is correct. I have noted this down and will work on this as a separate PR.